examples/classic_controllers/classic_controllers_synch_motor_test.py

```diff
@@ -3,95 +3,50 @@
 import gym_electric_motor as gem
```

Comment:
If this file should be added to the nightly/master branch, it should be tidied up a little